qlty-check/src/planner/config.rs

```diff
@@ -54,12 +48,26 @@ fn configure_plugins(planner: &Planner) -> Result<Vec<ActivePlugin>> {
             continue;
         }
 
-        if PLATFORM_DISABLED_PLUGINS.contains(&enabled_plugin.name.as_str()) {
-            debug!(
-                "Plugin {} is disabled on this platform, skipping.",
-                enabled_plugin.name
-            );
-            continue;
+        let current_platform = if cfg!(target_os = "windows") {
```

Comment:
I would extract this out into a dedicated function to encapsulate the cfg! stuff